src/modules/tests/tests.reducer.ts

```diff
@@ -20,11 +22,13 @@ export interface CurrentTest {
 export interface TestsModel {
   currentTest: CurrentTest;
   startedTests: { [slotId: string]: StandardCarTestCATBSchema };
+  testLifecycles: { [slotId: string]: TestStatus };
```

Comment:
Are there benefits of keeping track of test's life cycle outside of the startedTest[<SlotId>] collection?   
Maybe it would be better to introduce the test's state into the schema (for all categories)?

Reply:
The reason I excluded from `startedTests` is that this seemed to be tracking information specifically for the mobile app - I didn't see a reason that this should be submitted to the backend (which would imply it's part of the test schema). Do you think we would submit this status?

Reply:
Going to merge as-is for now, if we want to extend the test schema and refactor this state into there, I can do it later in this story.